server/src/domain/asset/asset.service.spec.ts

```diff
@@ -330,6 +330,56 @@ describe(AssetService.name, () => {
     });
   });
 
+  describe('getTimeBuckets', () => {
```

Comment:
Can you assert assetMock.getTimeBuckets are called with the right arguments?

Reply:
I thought about this but choose not to, because the dto object is just passed to getTimeBuckets without any intermediate steps. But sure, I can add this.

Reply:
Specifically if neither `albumId` nor `userId` are sent, the repository should be called with the `{ userId:  authUser.id, ... }`. If either one of those is present the `authUser.id` should be ignored and now show up in the call.